src/sphinxcontrib/towncrier/_towncrier.py

```diff
@@ -22,9 +22,13 @@ def get_towncrier_config(
         final_config_path: Path,
 ) -> Dict[str, Any]:  # FIXME: add a better type  # pylint: disable=fixme
     """Return the towncrier config dictionary."""
-    try:
+    try:  # noqa: WPS229
```

Comment:
Ignores are only acceptable in special cases. There's no good reason for adding this one. Just follow the rule guidance.